test.js

```diff
@@ -206,6 +217,26 @@ a[id="foo"] { content: "x"; }
         });
 });
 
+test('format scss uses the right parser', (t) => {
+    const filePath = './fixtures/trailing-comma/scss.scss';
+    const source = fs.readFileSync(filePath, 'utf8');
+
+    return format({
+        text: source,
+        filePath: filePath
+    }).then((source) => {
+        t.is(source,
+            `$greek: (
```

Comment:
The tabbing here looks a bit wonky. Is that intentional?

Reply:
Yep. 4 spaces on the first line (per the rest of the file), then within the backticks it has to match the prettier output which looks like:
```
$greek: (
alpha: 100,
beta: 200,
gamma: 300,
);
```

If i indented those lines then the comparison with the prettier output would fail.

I could jam all this on the same line if you think it makes it clearer:

```js
t.is(source, '$greek: (\nalpha: 100,\nbeta: 200,\ngamma: 300,\n);\n');
```

Other comparisons in this file use the backticks and multi-line approach but they're a little longer than this testcase.